cpp/tests/CMakeLists.txt

```diff
@@ -268,6 +268,15 @@ if (BUILD_MPI)
     ConfigureTest(NCCL_DEGREE_TEST "${NCCL_DEGREE_TEST_SRC}" "")
 endif(BUILD_MPI)
 
+###################################################################################################
```

Comment:
Do you mind deleting the dead code above in the `if (BUILD_MPI)` while editing this CMake file, please?

Reply:
thanks!